src/lib/tracking.ts

```diff
@@ -40,3 +40,20 @@ export const trackEvent: TrackEvent = ({ category, action, label, value }) => {
     })
   }
 }
+
+export const changeOperationAlias = (operation: string): string => {
```

Comment:
@fwaisi , since you are still working on that, please use enum CHANGE_OPERATION for this